src/doc/trpl/variable-bindings.md

```diff
@@ -11,7 +11,7 @@ fn main() {
 
 Putting `fn main() {` in each example is a bit tedious, so we’ll leave that out
 in the future. If you’re following along, make sure to edit your `main()`
-function, rather than leaving it off. Otherwise, you’ll get an error.
+function.
```

Comment:
"rather than leaving it off" is important info. I would re-write the sentence thus:

> If you're following along, ensure that you don't leave it off. (end of paragraph)

While at it it, I would change "in the future" to "from now on".